### PR TITLE
Enahncement/renamed request analyzer methods

### LINKED ISSUE #170: [RFC] Renamings of request analyzer methods
https://github.com/sulu-cmf/sulu/blob/feature/sulu-build/src/Sulu/Component/Webspace/Analyzer/RequestAnalyzer.php

- For all methods `Current` prefix is redundant (?) and implies that there is another state (i.e. what would getPortal return if not the "current" portal?)

- `getResource[Locator|LocatorPrefix]` => `getResourcePath`: Because `Locator` is a typical class name used by Locator classes, whilst `Path` is the most commonly used word to describe the "path" of a node within a repository.

### PR DESCRIPTION
**Pull request by dantleech**
This PR renames all of the methods in the RequestAnalyzerInterface which begin with `getCurrent`.

Fixes #170

### CODE REVIEW

File: src/Sulu/Component/Webspace/Analyzer/AdminRequestAnalyzer.php [hunk 1]
```diff
@@ -179,7 +179,7 @@ public function getCurrentPostParameter()
      * Returns the get parameters
      * @return array
      */
-    public function getCurrentGetParameter()
+    public function getGetParameters()
```
Comment (dantleech) [inline src/Sulu/Component/Webspace/Analyzer/AdminRequestAnalyzer.php]: Why do we do this? It is wrong

These parameters are only used by the ExceptionHandler as far as I can tell.

**Comment by alexander-schranz** [comment]
Please add this change to UPGRADE.md requestAnalyzer is used in some projects in the Client services/controllers

**Comment by dantleech** [comment]
ack

**Comment by dantleech** [comment]
Added CHANGELOG and UPGRADE. Ready to merge /cc @chirimoya

**Comment by dantleech** [comment]
@wachterjohannes @danrot can you approve this? then I will rebase and merge. (the test failure is due to the contact tests, not related).

**Comment by wachterjohannes** [comment]
@dantleech i think someone of use should review it and merge it after release on monday!

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,8 +2,10 @@ CHANGELOG for Sulu
 ==================
 
 * dev-develop
+
```
Comment (danrot) [inline CHANGELOG.md]: we don't put new lines in these lists

**Comment by danrot** [comment]
If the tests are running after the rebase it is fine for me.

**Pull request merged**